src/selectors/publish.js

```diff
@@ -117,18 +117,18 @@ export const getRemoveProfileInformation: Selector<RemoveProfileInformation | nu
     }
 
     return {
-      shouldFilterToCommittedRange: checkedSharingOptions.timeRange
-        ? committedRange
-        : null,
-      shouldRemoveNetworkUrls: checkedSharingOptions.urls,
-      shouldRemoveAllUrls: checkedSharingOptions.urls,
+      shouldFilterToCommittedRange: checkedSharingOptions.includeFullTimeRange
+        ? null
+        : committedRange,
+      shouldRemoveNetworkUrls: !checkedSharingOptions.includeUrls,
+      shouldRemoveAllUrls: !checkedSharingOptions.includeUrls,
```

Comment:
I didn't know that we consolidated those two. We can probably consolidate `shouldRemoveNetworkUrls` and `shouldRemoveAllUrls` in another PR.

Reply:
Good idea, I filed: https://github.com/firefox-devtools/profiler/issues/1941